pkg/api/schema/schema.proto

```diff
@@ -525,4 +566,33 @@ service ImmuService {
 	//			body: "*"
 	//		};
 	//	}
+	rpc CreateDatabase(Database) returns (CreateDatabaseReply) {
+		option (google.api.http) = {
+			post: "/v1/immurestproxy/createdatabase"
+			body: "*"
+		};
+	}
+	rpc UseDatabase(Database) returns (Error) {
+		option (google.api.http) = {
+			get: "/v1/immurestproxy/usedatabase/{databasename}"
+		};
+	}
+	rpc ChangePermission(ChangePermissionRequest) returns (Error) {
+		option (google.api.http) = {
+			post: "/v1/immurestproxy/changepermission"
+			body: "*"
+		};
+	}
+	rpc SetActiveUser (SetActiveUserRequest) returns (google.protobuf.Empty){
+		option (google.api.http) = {
+			post: "/v1/immurestproxy/user/SetActiveUser"
+			body: "*"
+		};
+	};
+	rpc DatabaseList (google.protobuf.Empty) returns (DatabaseListResponse){
+		option (google.api.http) = {
+			post: "/v1/immurestproxy/user/DatabaseList"
+			body: "*"
+		};
```

Comment:
@padurean @Gjergj Regarding endpoint my idea is to use more REST approach (remember to use all lower case chars):
```
CreateDatabase(Database) -> POST /v1/immurestproxy/database
UseDatabase(Database) -> POST /v1/immurestproxy/database/use
ChangePermission(ChangePermissionRequest) -> POST /v1/immurestproxy/database/permission
SetActiveUser (SetActiveUserRequest) -> POST /v1/immurestproxy/database/user/active
DatabaseList -> GET /v1/immurestproxy/database/user
```
What's your opinion?

Reply:
Agree, changed to lowercase